src/routes/auth/facebook.js

```diff
@@ -0,0 +1,123 @@
+import express  from 'express';
+import passport from 'passport';
+import { Strategy } from 'passport-facebook';
+
+import config   from '../../config';
+import db       from '../../models/db';
+import auth     from '../../middlewares/auth';
+import { finalizeAuth } from './utils';
+import {
+  ApiError,
+  BAD_REQUEST, ERRNO_BAD_REQUEST,
+  FORBIDDEN, ERRNO_FORBIDDEN,
+  UNAUTHORIZED, ERRNO_UNAUTHORIZED,
+} from '../../errors';
+
+const router = express.Router();
+
+const callbackURL =
+  `${config.get('publicHost')}/${config.get('version')}/auth/facebook/callback`;
```

Comment:
you mean the `callback` part ? In that case we can make `callback` the default ?

Just wondering: why do you think it's needed ?